_studio/mfx_lib/encode_hw/h265/include/mfx_h265_encode_hw_utils.h

```diff
@@ -596,11 +571,11 @@ namespace ExtBuffer
         bool bUnsuppoted = false;
         T buf_ref;
         Init(buf_ref);
-        CopySupportedParams(buf_ref, buf);
```

Comment:
you can't get rid of this one. you don't know which template function will be called